### PR TITLE
Navigation bar and poppins update

### LINKED ISSUE #12: Add CSS to the NavBar
-add style to navbar

### PR DESCRIPTION
**Pull request by MrGDS2**
-added in poppins text
-added icon to navbar(check esther)
-added nav link to home etc and bookings

### CODE REVIEW

File: madgiraffestudios/src/Fonts/Poppins/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2020 The Poppins Project Authors (https://github.com/itfoundry/Poppins)
```
Comment (estherkoko) [inline madgiraffestudios/src/Fonts/Poppins/OFL.txt]: What is this file for? We have poppins imported in the project in index.html and imported in index.css.

File: madgiraffestudios/src/index.css [hunk 2]
```diff
@@ -1,6 +1,9 @@
 body {
```
Comment (estherkoko) [inline madgiraffestudios/src/index.css]: any reason why this was reverted and changed to apple system .... font? we agreed to use google poppins font

File: madgiraffestudios/src/components/Navigation/Navigation.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.navbar-expand-lg .navbar-collapse {
+    font-family: 'Poppins' !important;
+    
+  }
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Navigation/Navigation.css]: please remove, poppins should be for the entire project - this way we don't need to set it here for navigation only

**Review by estherkoko — CHANGES_REQUESTED** [review]
Please remove the binary files for poppins and have a look at my comments. I will pull your branch and see how the changes are reflected

File: madgiraffestudios/src/components/Navigation/Navigation.css (hunk 3 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/Navigation/Navigation.css]: addressed

File: madgiraffestudios/src/Fonts/Poppins/OFL.txt (hunk 1 above)
Comment (MrGDS2) [reply madgiraffestudios/src/Fonts/Poppins/OFL.txt]: addressed

File: madgiraffestudios/src/App.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (estherkoko) [inline madgiraffestudios/src/App.scss]: @MrGDS2 this is causing failures

- [ ]  --can you remove the commented lines

File: madgiraffestudios/src/Fonts/Poppins/OFL.txt (hunk 1 above)
Comment (estherkoko) [inline madgiraffestudios/src/Fonts/Poppins/OFL.txt]: - [ ] please remove this file and all binary files

File: madgiraffestudios/src/components/Navigation/NavigationBar.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import giraffeicon from '../../Images/icon.png';
+
+
+const NavigationBar = () => (
+
+<div className="container">
+    <Navbar expand="lg">
+        <Navbar.Brand href="/home"><img src={giraffeicon} height="50px" width="50px" /></Navbar.Brand>
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Navigation/NavigationBar.js:13]: consider using reactstrap here instead - also, adding ml and mr will increase css formatting for mobile devices/view.  

- [x] Consider using spacing utilities  more like mx/px values instead
- [x] add pt-3 to the container so it's not too close to the browser search bar
- [x] make font-size bigger, add this to app.scss  | $font-size-base: 0.9rem !default; |
This way all text for the project will inherit this size by default unless any specific styling is required for an element.

File: madgiraffestudios/src/index.js [hunk 6]
```diff
@@ -1,9 +1,29 @@
...
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import './index.css';
 import App from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import * as serviceWorker from './serviceWorker';
+import NavigationBar from './components/Navigation/NavigationBar'
+//import About from './components/About/About';
+import { WORK } from './Constants/routes';
 
+const routing = (
```
Comment (estherkoko) [inline madgiraffestudios/src/index.js:12]: - [x] please remove extra line spaces and commented code

File: madgiraffestudios/src/App.js [hunk 7]
```diff
@@ -1,26 +1,14 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+  
+      <NavigationBar />
+      // <Header/>
```
Comment (estherkoko) [inline madgiraffestudios/src/App.js]: - [x] please remove commented lines

File: madgiraffestudios/src/index.js [hunk 8]
```diff
@@ -1,9 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import './index.css';
 import App from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (estherkoko) [inline madgiraffestudios/src/index.js]: - [ ] don't think this belong here or is the proper way to import bootstrap, please check
- [ ]  move bootstrap import into app.scss - should look similar to this: 
@import "~bootstrap/scss/bootstrap"; If it's already in app.scss, then just delete this import here

File: madgiraffestudios/src/components/Navigation/NavigationBar.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Navigation/NavigationBar.js:3]: - [x] use reactstrap here - import should look something like below but google reactstrap to confirm:
import { Navbar, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap';

**Review by estherkoko — CHANGES_REQUESTED** [review]

File: madgiraffestudios/src/components/Navigation/NavigationBar.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+import giraffeicon from '../../Images/icon.png';
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Navigation/NavigationBar.js:4]: - [ ] use absolute path here

File: madgiraffestudios/package.json [hunk 11]
```diff
@@ -6,8 +6,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
```
Comment (estherkoko) [inline madgiraffestudios/package.json:7]: Beautiful 🥇

**Review by estherkoko — CHANGES_REQUESTED** [review]
- [ ] Please address task I listed, thanks

File: madgiraffestudios/src/App.scss (hunk 4 above)
Comment (MrGDS2) [reply madgiraffestudios/src/App.scss]: resolved

File: madgiraffestudios/src/App.js (hunk 7 above)
Comment (MrGDS2) [reply madgiraffestudios/src/App.js]: good to go

File: madgiraffestudios/src/index.js (hunk 6 above)
Comment (MrGDS2) [reply madgiraffestudios/src/index.js:12]: spacing fixed

File: madgiraffestudios/src/components/Navigation/NavigationBar.js (hunk 5 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/Navigation/NavigationBar.js:13]: addressed

File: madgiraffestudios/src/components/Navigation/NavigationBar.js (hunk 10 above)
Comment (estherkoko) [reply madgiraffestudios/src/components/Navigation/NavigationBar.js:4]: > * [ ]  use absolute path here

I will help address this

**Review by estherkoko — APPROVED** [review]

**Comment by estherkoko** [comment]
Great work on your first big PR 👍

**Pull request merged**